ipv-api/build.gradle

```diff
@@ -28,14 +28,16 @@ dependencies {
             configurations.lambda,
             configurations.sqs,
             configurations.kms,
-            configurations.dynamodb
+            configurations.dynamodb,
+            "au.com.dius.pact.consumer:junit5:4.6.4"
```

Comment:
Let's stick to the established pattern here

Reply:
Do we need/want to do anything about getting it onto artifactory ?

Reply:
Nope, artifactory is just a transparent cache at the moment